.github/workflows/publish_docker.yaml

```diff
@@ -39,7 +39,15 @@ jobs:
         run: |
           if [[ ${{ github.event.ref }} =~ ^refs/tags/[0-9]+\.[0-9]+\.[0-9]+$ ]]; then
               echo ::set-output name=match::true
+          else
+              echo ::set-output name=match::false
```

Comment:
that's true, fair point. but i'll put this check/filter within the nox session rather than here. who wants to write bash code embedded in yaml 😒

Reply:
this actually made me realize i also forgot to `return` on the case of not finding any current tag, which is important. i've added that [here](https://github.com/ethyca/fides/pull/3050/files#diff-4891be4bd3dfdc5bb907500380ae2cb348070fb9b49d676259fc13b5b3174ae5R200) and also added a check [here](https://github.com/ethyca/fides/pull/3050/files#diff-4891be4bd3dfdc5bb907500380ae2cb348070fb9b49d676259fc13b5b3174ae5R202) to make sure we _recognize_ the tag pattern.

it seems like it'll be hard for me to test the negative case, my attempts at tagging with bogus tags are preventing me from even running the "build" session locally since versioneer fails. so i think we'll probably fail in CI before we even get to this point. but i think it's still a good check to have in place, and i can verify the positive case, i.e. that our recognized tag types continue to successfully publish git-tagged images.